actions/tag-and-release/action.yaml

```diff
@@ -0,0 +1,44 @@
+name: "Tag and release"
+description: "Tag and release with target tag."
+
+inputs:
+  tag:
+    description: "The version of the tag to be publish and released, e.g. 1.0.0."
+    required: true
+  release-title:
+    description: "Title for the GitHub release."
+    required: true
+  release-body:
+    description: "Description for the GitHub release."
+    default: ""
+    required: false
+  github-username:
+    description: "The GitHub username for committing the changes."
+    required: true
+  github-email:
+    description: "The GitHub email for committing the changes."
+    required: true
+  github-token:
+    description: "The GitHub token for committing the changes."
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Create tag
+      run: |
+        git config --local user.name ${{ inputs.github-username }}
+        git config --local user.email ${{ inputs.github-username }}
+        git tag ${{ inputs.tag }}
```

Comment:
Where is this tag pushed?

Reply:
As I understand. It gets pushed over the release. It gets created and pushed. You can see it on the playground